setup.py

```diff
@@ -52,8 +52,8 @@
         "rl "
         "reinforcement learning "
     ),
-    install_requires=["gym", "ai-safety-gridworlds", "numpy==1.14.5"],
-    dependency_links=["https://github.com/jvmancuso/ai-safety-gridworlds/"],
+    install_requires=["gym", "ai-safety-gridworlds", "numpy>=1.14.5"],
```

Comment:
I chose the exact version, because that is what they give as a dependency in the `ai-safety-gridworlds` repo. They say "Numpy. Our version is 1.14.5. Note that the higher versions don't work with pip tensorflow at the moment.". Not sure if this is still a problem, or can we safely use higher versions?